packages/scene-composer/src/components/panels/scene-components/tag-style/ColorPicker/ColorPicker.tsx

```diff
@@ -70,12 +72,15 @@ export const ColorPicker = ({
     };
   }, [showPicker, handleOutsideClick]);
 
-  const checkIfCustomColor = (color): void => {
-    if (!Object.values(palleteColors).includes(color)) {
-      setCustomInternalColors(customInternalColors.concat(color));
-      onUpdateCustomColors?.([...new Set(customInternalColors)]);
-    }
-  };
+  const checkIfCustomColor = useCallback(
+    (color): void => {
+      if (!Object.values(palleteColors).includes(color) && color !== colors.infoBlue) {
+        setCustomInternalColors(customInternalColors.concat(color));
+        onUpdateCustomColors?.([...new Set(customInternalColors)]);
```

Comment:
`customInternalColors` here is not updated to have the new color yet since setState is async. you should have a local array with the new value